html/elements/script.json

```diff
@@ -548,6 +548,224 @@
                 "deprecated": false
               }
             }
+          },
+          "speculationrules": {
+            "__compat": {
+              "spec_url": "https://wicg.github.io/nav-speculation/speculation-rules.html#speculation-rules-script",
+              "support": {
+                "chrome": {
+                  "version_added": "105",
+                  "notes": "Initial support included same-origin prerendering only."
+                },
+                "chrome_android": {
+                  "version_added": "103",
+                  "notes": "Initial support included same-origin prerendering only."
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": "mirror",
+                "samsunginternet_android": "mirror",
+                "webview_android": "mirror"
+              },
+              "status": {
+                "experimental": true,
+                "standard_track": true,
+                "deprecated": false
+              }
+            },
+            "crossorigin_prerending": {
```

Comment:
This should be removed and be an inline version range for Chrome.

Reply:
Done